server/services/message-service.js

```diff
@@ -1,38 +1,32 @@
-const amqp = require('amqplib/callback_api')
-const scheduleService = require('./schedule-service')
+const container = require('rhea')
 const config = require('../config')
+const scheduleService = require('./schedule-service')
 
 module.exports = {
   receiveSchedule: function () {
-    const messageQueue = config.messageQueue
-    amqp.connect(messageQueue, function (err, conn) {
-      if (err) {
-        console.log(err)
-      } else {
-        conn.createChannel(function (err, ch) {
-          if (err) {
-            console.log(err)
-          } else {
-            const scheduleQueue = 'schedule'
-            ch.assertQueue(scheduleQueue, { durable: false })
-            console.log('waiting for new claims')
-
-            ch.consume(scheduleQueue, function (msg) {
-              console.log(`claim received for scheduling - ${msg.content.toString()}`)
-              scheduleService.create(JSON.parse(msg.content))
-            }, { noAck: true })
-
-            const valueQueue = 'value'
-            ch.assertQueue(valueQueue, { durable: false })
-            console.log('waiting for new calculation')
-
-            ch.consume(valueQueue, function (msg) {
-              console.log(`calculation received for payment - ${msg.content.toString()}`)
-              scheduleService.updateValue(JSON.parse(msg.content))
-            }, { noAck: true })
-          }
-        })
-      }
+    const scheduleQueue = 'schedule'
+    const valueQueue = 'value'
+    const activeMQOptions = {
+      transport: config.messageQueuePort === 5672 ? 'tcp' : 'ssl',
+      port: config.messageQueuePort,
+      reconnect_limit: 10,
+      host: config.messageQueue,
+      hostname: config.messageQueue,
+      username: config.messageQueueUser,
+      password: config.messageQueuePass
+    }

```

Comment:
Are both `host` and `hostname` required in the message queue configuration?

The activeMQOptions object is created for each call to the function, even though it doesn't change. It could be moved to a constant at the top of the file.

The message queue options are defined in top level fields the config object.
it may be cleaner to define a separate Joi schema and config for the message queue options, or have a messageQueue subobject on `config` with the fields transport, port, etc that can be required in the message-service, i.e. 
`const activeMQOptions = require('config').messageQueue`